v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/exception/PrimaryKeyNotFoundException.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.source.reader.io.exception;
```

Comment:
Please subclass this under v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/exception/SchemaDiscoveryException.java

Reply:
It is already there. Please look at the class declaration.